dbfit-java/informix/src/main/java/dbfit/environment/InformixEnvironment.java

```diff
@@ -0,0 +1,338 @@
+        qry += "                                               WHEN  0 THEN 'YEAR'";
+        qry += "                                               WHEN  2 THEN 'MONTH'";
+        qry += "                                               WHEN  4 THEN 'DAY'";
+        qry += "                                               WHEN  6 THEN 'HOUR'";
+        qry += "                                               WHEN  8 THEN 'MINUTE'";
+        qry += "                                               WHEN 10 THEN 'SECOND'";
+        qry += "                                               WHEN 11 THEN 'FRACTION(1)'";
+        qry += "                                               WHEN 12 THEN 'FRACTION(2)'";
+        qry += "                                               WHEN 13 THEN 'FRACTION(3)'";
+        qry += "                                               WHEN 14 THEN 'FRACTION(4)'";
+        qry += "                                               WHEN 15 THEN 'FRACTION(5)'";
+        qry += "                                           END";
+        qry += "            WHEN   11 THEN 'BYTE'";
+        qry += "            WHEN   12 THEN 'TEXT'";
+        qry += "            WHEN   13 THEN 'VARCHAR'";
+        qry += "            WHEN   14 THEN 'INTERVAL'";
+        qry += "            WHEN   15 THEN 'NCHAR'";
+        qry += "            WHEN   16 THEN 'NVARCHAR'";
+        qry += "            WHEN   17 THEN 'INT8'";
+        qry += "            WHEN   18 THEN 'SERIAL8'";
+        qry += "            WHEN   19 THEN 'SET'";
+        qry += "            WHEN   20 THEN 'MULTISET'";
+        qry += "            WHEN   21 THEN 'LIST'";
+        qry += "            WHEN   22 THEN 'ROW'";
+        qry += "            WHEN   23 THEN 'COLLECTION'";
+        qry += "            WHEN   24 THEN 'ROWDEF'";
+        qry += "            WHEN   40 THEN 'LVARCHAR'";
+        qry += "            WHEN   43 THEN 'LVARCHAR'";
+        qry += "            WHEN   45 THEN 'BOOLEAN'";
+        qry += "            WHEN   52 THEN 'BIGINT'";
+        qry += "            WHEN   53 THEN 'BIGSERIAL'";
+        qry += "            WHEN 2061 THEN 'IDSSECURITYLABEL'";
+        qry += "            WHEN 4118 THEN 'ROW'";
+        qry += "        END";
+        qry += "           AS data_type";
+        qry += "     , " + (forProcedureParameters ? "c.paramattr" : "1"); // Dummy value for table/view columns.
+        qry += "           AS direction";
+        qry += "     , c." + (forProcedureParameters ? "paramid" : "colno");
+        qry += "           AS position";
+        qry += "  FROM informix." + (forProcedureParameters ? "sysprocedures p" : "systables p");
+        qry += "     , informix." + (forProcedureParameters ? "sysproccolumns c" : "syscolumns c");
+        qry += " WHERE p." + (forProcedureParameters ? "procid" : "tabid") + " = c." + (forProcedureParameters ? "procid" : "tabid");
+        qry += "   AND ";
```

Comment:
What about concatenating these strings while initializing the string (instead of creating appended versions multiple times with `+=`)?